### DISCUSSION TITLE
Will you implement image transforms on DALI or in GPU?

### DISCUSSION THREAD

**Discussion by ZhongkangLu** [Project-MONAI/MONAI#4496] [Ideas]
Hi,

I am working on 3D MRI image segmentation. The augmentation in CPU is quite slow and my GPUs have to wait for it. Will you implement image transforms in GPU? Or base on the tools such as Nvidia DALI?

Thanks

**Comment by Nic-Ma** [comment]
Hi @ZhongkangLu ,

Most of the MONAI transforms can execute on GPU directly, please check this documentation and the tutorial:
https://github.com/Project-MONAI/tutorials/blob/main/acceleration/fast_model_training_guide.md#2-execute-transforms-on-gpu

Thanks.

**Reply by ZhongkangLu** [reply]
Thank you fr your prompt response! Really appreciated.
